packages/realm/package.json

```diff
@@ -264,7 +264,7 @@
     "prebuild-install": "^7.1.1"
   },
   "peerDependencies": {
-    "bson": "^4",
+    "bson": "^5 || ^4",
```

Comment:
[v11](https://github.com/realm/realm-js/blob/49f3d84a5323347ee20749b09b363a9320c00ea9/package.json#L87) is pinned to `4.4.1`.

Reply:
Good catch! ⚾

Reply:
I pinned it to 4.4.1 since 4.5.0 broke on React Native.

Reply:
Thanks for clarification!